simple-api/README.md

```diff
@@ -0,0 +1,2 @@
+# Simple API Server
```

Comment:
lets add more context here, on what might make this useful for the user. Why does it have a kurtosis package within it and such

Reply:
Yes you are right. I'll do that